src/io/flutter/logging/FlutterConsoleLogManager.java

```diff
@@ -40,134 +43,197 @@
   }
 
-  private static final int COL_WIDTH = 102;
-
   private void printDiagnosticsNodeProperty(ConsoleView console, String indent, DiagnosticsNode property,
-                                            ConsoleViewContentType contentType) {
-    if (contentType == null) {
-      contentType = SUBTLE_CONTENT_TYPE;
+                                            ConsoleViewContentType contentType,
+                                            boolean isInChild) {
+    // TODO(devoncarew): Change the error message display in the framework.
+    if (property.getDescription() != null && property.getLevel() == DiagnosticLevel.info) {
+      // Elide framework '◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤...' lines.
+      if (property.getDescription().startsWith("◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤")) {
+        return;
+      }
+
+      // Elide framework blank styling lines.
+      if (StringUtil.equals("ErrorSpacer", property.getType())) {
+        return;
+      }
+    }
 
-      if (property.getLevel().compareTo(DiagnosticLevel.error) >= 0) {
-        contentType = ConsoleViewContentType.ERROR_OUTPUT;
+    if (contentType == null) {
+      if (property.getLevel() == DiagnosticLevel.error) {
+        contentType = ERROR_CONTENT_TYPE;
+      }
+      else if (property.getLevel() == DiagnosticLevel.summary) {
+        contentType = ERROR_CONTENT_TYPE; // NORMAL_CONTENT_TYPE;
       }
-      else if (property.getLevel().compareTo(DiagnosticLevel.summary) >= 0) {
-        contentType = ConsoleViewContentType.NORMAL_OUTPUT;
+      else if (property.getLevel() == DiagnosticLevel.hint) {
+        contentType = NORMAL_CONTENT_TYPE;
+      }
+      else {
+        contentType = SUBTLE_CONTENT_TYPE;
       }
     }
 
     console.print(indent, contentType);
 
-    int col = 0;
     if (property.getShowName()) {
       console.print(property.getName(), contentType);
-      col += property.getName().length();
     }
     if (property.getShowName() && property.getShowSeparator()) {
       console.print(property.getSeparator() + " ", contentType);
-      col += property.getSeparator().length() + 1;
     }
 
-    // Wrap long descriptions.
+    // Print the description.
     if (property.getDescription() != null) {
       final String description = property.getDescription();
+      console.print(description + "\n", contentType);
+    }
+    else {
+      console.print("\n", contentType);
+    }
 
-      if (property.getAllowWrap()) {
-        final String[] lines = wrap(description, COL_WIDTH - indent.length(), col);
+    if (property.hasChildren()) {
+      try {
+        final CompletableFuture<ArrayList<DiagnosticsNode>> future = property.getChildren();
+        final ArrayList<DiagnosticsNode> children = future.get();
 
-        if (lines.length == 1) {
-          console.print(lines[0] + "\n", contentType);
+        if (!isInChild && children.stream().noneMatch(DiagnosticsNode::hasChildren)) {
+          final String childIndent = indent + "  ";
```

Comment:
if the parent has style DiagnosticsTreeStyle.flat 
then the child indent should just be the indent. Somehow your indents in the screenshot seem flat for the outer children. Not sure how that is happening.
https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/foundation/diagnostics.dart#L1674

Reply:
OK, I'll adjust the indent when I see `DiagnosticsTreeStyle.flat`. The initial set of children - the immediate properties on the root diagnostics node - do have some special casing in terms of indent and padding blank lines.

Reply:
Things like exception traces now print out with less indent (seems legit):

<img width="1085" alt="Screen Shot 2019-07-10 at 4 21 46 PM" src="https://user-images.githubusercontent.com/1269969/61011318-ea863d80-a32e-11e9-87d2-016e17cf76c5.png">